packages/spark-core/tests/spinners.tests.js

```diff
@@ -0,0 +1,84 @@
+/* global document beforeEach describe it */
```

Comment:
I didn't see an example of the spinner click event on the test page or the drizzle docs page. Might be worthwhile to add it in so apps could see the supplied spinner JS in action.

Reply:
the only one is on the buttons test page, for submission spinner